diff --git a/node_modules/@babel/helper-compilation-targets/lib/options.js b/node_modules/@babel/helper-compilation-targets/lib/options.js
index fcabd96..cbf4de0 100644
--- a/node_modules/@babel/helper-compilation-targets/lib/options.js
+++ b/node_modules/@babel/helper-compilation-targets/lib/options.js
@@ -15,6 +15,7 @@ const TargetNames = {
   ios: "ios",
   android: "android",
   electron: "electron",
-  samsung: "samsung"
+  samsung: "samsung",
+  rhino: "rhino"
 };
 exports.TargetNames = TargetNames;
\ No newline at end of file
